static/bidder-params/roulax.json

```diff
@@ -0,0 +1,32 @@
+{
+  "$schema": "http://json-schema.org/draft-04/schema#",
+  "title": "Roulax Adapter Params",
+  "description": "A schema which validates params accepted by the Roulax adapter",
+  "type": "object",
+
+  "properties": {
+    "placementId": {
+      "type": "string",
+      "description": "Placement ID"
+    }
+  },
+
+  "properties": {
+    "PId": {
+      "type": "string",
+      "description": "PID"
+    }
+  },
+
+  "properties": {
+    "PublisherPath": {
+      "type": "string",
+      "description": "PublisherPath"
+    }
+  },
```

Comment:
Should combine this under one parent properties. Example - 
https://github.com/prebid/prebid-server/blob/master/static/bidder-params/appush.json#L10